beagle_etl/views.py

```diff
@@ -77,6 +77,14 @@ def fix_query_list(self, request_query, key_list):
                     query_dict[single_param] = query_value
         return query_dict
 
+class AssayViewSet(mixins.CreateModelMixin,
+                 mixins.DestroyModelMixin,
```

Comment:
Assay is basically model which is used as configuration, and we shouldn't have Create and Destroy endpoints. We only need to get values from that record, and update it, we might implement this as a GET, and POST `/v0/assay` where we will return values on GET, and update them on POST. We should also add validation on POST, that all assays specified in `disabled` and `hold`, needs to be listed in `all` as well, and same assay shouldn't be in both `disabled` and `hold`.